src/content/docs/alerts/get-notified/muting-rules-suppress-notifications.mdx

```diff
@@ -245,91 +245,117 @@ A muting rule has the following fields and components:
 
-* <DNT>**Fully muted (`FULLY_MUTED`)**</DNT>: an issue has all of its open incidents muted (Default value).
-* <DNT>**Partially muted (`PARTIALLY_MUTED`)**</DNT>: an issue that has at least one open incident that is muted and one open incident that is not muted.
-* <DNT>**Not muted (`NOT_MUTED`)**</DNT>: an issue that has no open muted incidents.
+Rules can have one of the following statuses:
 
-For a step-by-step guide on how to set up your workflows, check out an example demo below (approx. 2:17 minutes):
+* <DNT>**Active**</DNT>: Muting is enabled and active.
+* <DNT>**Scheduled**</DNT>: Muting is enabled but not active yet (there's a future schedule).
+* <DNT>**Ended**</DNT>: Muting is enabled, but no longer active (there's no future schedule).
+* <DNT>**Inactive**</DNT>: Muting is disabled.
 
-<Video
-  type="wistia"
-  id="3nf3q2supe"
+<img
+  title="Manage muting rules"
+  alt="Manage muting rules"
+  src="/images/alerts_screenshot-full_muting-rules.webp"
+/>
+
+<figcaption>
+  Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Muting rules**</DNT>: You can create complex muting rules to target a small or large set of unwanted notifications.
+</figcaption>
+
+## Create a muting rule [#create]
+
+<Callout variant="tip">
+  Before creating muting rules, you'll need to [create policies](/docs/alerts/new-relic-alerts/configuring-alert-policies/create-edit-or-find-alert-policy) and [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) that generate [notifications](/docs/alerts-applied-intelligence/notifications/intro-notifications/).
+</Callout>
+
+To create a muting rule, follow these steps:
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts**</DNT> and click <DNT>**Muting rules**</DNT> on the left navigation pane.
+
+2. Click <DNT>**+ Add a rule**</DNT>. 
+
+3. Enter a name and a description (optional) for the muting rule, and select the account to which the rule will apply.
+
+4. Build the incidents filter. You can use a subset of [incident event attributes](/docs/alerts/create-alert/condition-details/incident-event-attributes/). Choose an attribute, an [operator](#sub-conditions), and a value. These are the attributes: `accountId`, `conditionId`, `conditionName`, `conditionType`, `entity.guid`, `nrqlEventType`, `nrqlQuery`, `policyId`, `policyName`, `product`,`runbookUrl` (as `conditionRunbookUrl`), `tags.<NAME>`, and `targetName`). Values are compared against one of your incident attributes, such as an alerts policy ID or a condition name.
+
+5. Click <DNT>**Add another condition**</DNT> if you want to include more filters.
```

Comment:
MSTP suggestion: place > purpose > Action, so an alternate might be:

If you want to include more filters, click <DNT>**Add another condition**</DNT>